narzedzia/gatling

```diff
@@ -0,0 +1,45 @@
+# Gatling
+
+Gatling to biblioteka do testów wydajnościowych, którą można również stosować do testów funkcjonalnych. Możemy korzystać z całej gamy gotowych protokołów, jak również rozszerzać bibliotekę do własnych potrzeb, używajac języka Scala.
+
+### Strona projektu
+* [https://gatling.io/](https://gatling.io/)
+
+### Zestaw przygotowany przez autorów
+* [https://gatling.io/documentation/](https://gatling.io/documentation/)
+
+### Materiały do nauki Gatlinga
+* [https://www.baeldung.com/introduction-to-gatling](https://www.baeldung.com/introduction-to-gatling)
+* [https://www.swtestacademy.com/introduction-gatling/](https://www.swtestacademy.com/introduction-gatling/)
+
+### Materiały do nauki Scali
+* [https://twitter.github.io/scala_school/](https://twitter.github.io/scala_school/)
+ * Kurs Scali na Twitterze
+* [https://docs.scala-lang.org/tutorials/](https://docs.scala-lang.org/tutorials/)
+ * Oficjalna dokumentacja
+ 
+* [https://www.tutorialspoint.com/scala/](https://www.tutorialspoint.com/scala/)
+ * Tutorials point
+* [https://www.scala-exercises.org/](https://www.scala-exercises.org/)
+ * Scala - ćwiczenia
+* [http://allaboutscala.com/](http://allaboutscala.com/)
+ * Wszystko o Scali
+* [https://www.javatpoint.com/scala-tutorial](https://www.javatpoint.com/scala-tutorial)
+ * Java T Point
+* [Programming in Scala](https://www.enbook.pl/catalog/product/view/id/1616076?gclid=EAIaIQobChMImpGFjrvG3QIVTPlRCh1tdArOEAQYASABEgJalvD_BwE)
+ * Książka Martina Odersky'ego, autora Scali
+* [Programowanie funkcyjne. Krok po kroku](https://helion.pl/ksiazki/programowanie-funkcyjne-krok-po-kroku-joshua-backfield,pfukpk.htm#format/d)
+ * Książka o paradygmacie funcyjnym (jest tam również język Groovy)
+* [Scala od podszewki](https://helion.pl/ksiazki/scala-od-podszewki-joshua-suereth-d,scalao.htm#format/e)
+ * Książka o Scali po polsku
+
+### Kursy komercyjne
+* [http://testautomation.school/performance/](http://testautomation.school/performance/)
```

Comment:
Z agendy wynika, że co najwyżej 10% kursu jest skupione wokół gatlinga.
To jest podstrona narzędzia gatling nie podstrona testów wydajnościowych.
Do usunięcia lub przeniesienia na inną podstronę dedykowaną testowaniu wydajnościowemu i kursom z tym powiązanym.